### PR TITLE
convert address to token

### PR DESCRIPTION
**Pull request by wenjieyek**

### CODE REVIEW

File: lib/token_explorer.rb [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+      token.price = output['price']
+      token.price_change = output['price']['diff']
+      token.price_change_7d = output['price']['diff7d']
+      token.price_change_30d = output['price']['diff30d']
+      token.price_timestamp = Time.now.to_i
+      token.market_cap_usd = output['price']['marketCapUsd']
+      token.available_supply = output['price']['availableSupply']
+      token.volume_24h = output['price']['volume24h']
+      token.price_currency = output['price']['currency']
+      token(token)
```
Comment (tmlee) [inline lib/token_explorer.rb]: Can just return `token` without the need for method below

File: lib/token_explorer/helpers/numeric_helper.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class NumericHelper
+  class << self
+    def to_d(number)
```
Comment (tmlee) [inline lib/token_explorer/helpers/numeric_helper.rb]: @wenjieyek Rails has a `to_d` you should be able to use directly without needing to implement it

File: lib/token_explorer/helpers/numeric_helper.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        num = number.to_s
+        num.empty? ? nil : BigDecimal.new(num)
+      else
+        nil
+      end
+    rescue
+      nil
+    end
+
+    def divide(number, divisor)
```
Comment (tmlee) [inline lib/token_explorer/helpers/numeric_helper.rb]: `divide` and `to_numeric` not used, do we need them?

File: lib/token_explorer/helpers/numeric_helper.rb (hunk 2 above)
Comment (wenjieyek) [reply lib/token_explorer/helpers/numeric_helper.rb]: I think I just using to_d is ok

**Pull request merged**